src/main/java/www/purple/mixxy/dao/ComicDao.java

```diff
@@ -46,6 +46,10 @@ public Comic getComic(long id) {
 	public Comic getComic(User user) {
 		return objectify.get().load().type(Comic.class).filter("authorIds", user.id).first().now();
 	}
+	
+	public Comic getRemix(long id) {
```

Comment:
Is this supposed to be the parent of the given comic?  If so, I think `getParent` would be a better name.